app/services/invoices/preview_context_service.rb

```diff
@@ -88,6 +90,10 @@ def build_subscription
       )
     end
 
+    def find_or_build_subscriptions
```

Comment:
I'm not sure with the name, I don't see where it "find", it seems to always "build".
Maybe `build_subscriptions` (plural) is enough but you mihgt have other things in mind coming.

Reply:
Yep, sorry, maybe this PR is hard to track since it is just setup for multiple subscriptions and not a final implementation. That's why it goes in feature branch.
Currently we just added support for array but we don't really support multiple subscriptions that are passed as parameters.
In the next PRs we will extend this method and add logic where subscriptions can be either loaded from the DB (based on the external_ids from the params) or we will build subscription object as before